examples/parallel/plot_streaming_pipeline.py

```diff
@@ -0,0 +1,70 @@
+"""
+Parallel processing in Neuraxle
+===================================================================
+
+This demonstrates how to stream data in parallel in a Neuraxle pipeline.
+
+..
+    Copyright 2019, Neuraxio Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+
+"""
+import time
+import numpy as np
+
+from neuraxle.distributed.streaming import SequentialQueuedPipeline
+from neuraxle.pipeline import Pipeline
+from neuraxle.steps.loop import ForEachDataInput
+from neuraxle.steps.misc import Sleep
+from testing.test_streaming import MultiplyBy
+
+
+def main():
+    sleep_time = 0.02
+    p = SequentialQueuedPipeline([
+        Pipeline([ForEachDataInput(Sleep(sleep_time=sleep_time)), MultiplyBy(2)]),
+        Pipeline([ForEachDataInput(Sleep(sleep_time=sleep_time)), MultiplyBy(2)]),
+        Pipeline([ForEachDataInput(Sleep(sleep_time=sleep_time)), MultiplyBy(2)]),
+        Pipeline([ForEachDataInput(Sleep(sleep_time=sleep_time)), MultiplyBy(2)])
+    ], n_workers_per_step=8, max_size=10, batch_size=10)
```

Comment:
What does max_size means ? Maybe this argument is not named properly because it sounds unclear to me when reading this piece of code. Is it max_queue_size perhaps?

Reply:
the argument name is max size in the multiprocessing.Queue object. ill rename the argument on the SequentialQueuedPipeline